ckcp/docker/ckcp/Dockerfile

```diff
@@ -1,13 +1,13 @@
 #use golang as base image and compile kcp
```

Comment:
This file (and the ckcp folder) is an older version of self hosted kcp we were using earlier (before the shared instance came into picture). After the merge of #38, we plan to do a clean up of the repo.